src/server.js

```diff
@@ -48,6 +49,25 @@ module.exports = function() {
 			maxAge: 86400 * 1000,
 		}));
 
+	app.get("/uploads/:name/:slug*?", (req, res) => {
```

Comment:
https://stackoverflow.com/questions/6161567/express-js-wildcard-routing-to-cover-everything-under-and-including-a-path